lib/blockchain.go

```diff
@@ -2641,9 +2641,170 @@ func (bc *Blockchain) processBlockPoW(desoBlock *MsgDeSoBlock, verifySignatures
 // 4. Connect the block to the blockchain's tip
 // 5. If applicable, flush the incoming block's grandparent to the DB
 // 6. Notify the block proposer, pacemaker, and voting logic that the incoming block has been accepted
-func (bc *Blockchain) processBlockPoS(desoBlock *MsgDeSoBlock, verifySignatures bool) (_isMainChain bool, _isOrphan bool, err error) {
+func (bc *Blockchain) processBlockPoS(desoBlock *MsgDeSoBlock, verifySignatures bool) (_success bool, _isOrphan bool, _blockHashToRequest *BlockHash, _err error) {
```

Comment:
Went back and forth on this. Should we just make the `_blockHashToRequest` return value a slice of BlockHashes? Seems more flexible and a bit more future proof of we do. 

The node implementation to request the blocks won't change really.